dev/PagerControl/PagerControl.cpp

```diff
@@ -124,6 +124,8 @@ void PagerControl::OnApplyTemplate()
         m_comboBox.set(comboBox);
         if (comboBox)
         {
+            FillComboBoxCollectionToSize(NumberOfPages());
+            comboBox.UpdateLayout();
```

Comment:
I don't think manually calling UpdateLayout here is necessary. We're only concerned about the underlying item source which will be filled.

Reply:
If you say we don't need than let's leave it out.